celery_app/bc.txt

```diff
@@ -0,0 +1,105 @@
+# -*- encoding: utf-8 -*-
```

Comment:
are `celery_app/` and `celeryapp` 2 different packages you created?
if so why?

as the main intent is just introducing tasks into the project and not creating new package,
a module would be more preferable to  hold specific tasks


what is the `bc.txt` file being used for?